src/test/starknet.t.sol

```diff
@@ -36,7 +37,8 @@ contract ConfigStarknet {
             dai_bridge_isOpen:         1,        // 1 open, 0 closed
             dai_bridge_ceiling:        200_000,  // Whole Dai Units
             dai_bridge_maxDeposit:     1000,     // Whole Dai Units
-            l2_dai_bridge:             0x075ac198e734e289a6892baa8dd14b21095f13bf8401900f5349d5569c3f6e60
+            l2_dai_bridge:             0x075ac198e734e289a6892baa8dd14b21095f13bf8401900f5349d5569c3f6e60,
+            l2_gov_relay:              0x05f4d9b039f82e9a90125fb119ace0531f4936ff2a9a54a8598d49a4cd4bd6db
```

Comment:
where is the l2 deployed contract list to check against?

Reply:
> where is the l2 deployed contracts list to check against?

https://github.com/makerdao/starknet-dai-bridge/tree/wormhole#deployed-addresses

Reply:
`l2_gov_relay` doesn't match the linked one, have been updated?

Reply:
Not yet, as new gov relay is not yet authorised on l2. Old one still works.